src/main/java/org/spongepowered/asm/mixin/transformer/ClassInfo.java

```diff
@@ -489,19 +506,14 @@ private ClassInfo() {
         this.superName = null;
         this.outerName = null;
         this.isProbablyStatic = true;
-        this.methods = ImmutableSet.<Method>of(
-            new Method("getClass", "()Ljava/lang/Class;"),
-            new Method("hashCode", "()I"),
-            new Method("equals", "(Ljava/lang/Object;)Z"),
-            new Method("clone", "()Ljava/lang/Object;"),
-            new Method("toString", "()Ljava/lang/String;"),
-            new Method("notify", "()V"),
-            new Method("notifyAll", "()V"),
-            new Method("wait", "(J)V"),
-            new Method("wait", "(JI)V"),
-            new Method("wait", "()V"),
-            new Method("finalize", "()V")
-        );
+        this.methods = ImmutableSet.<Method>of(new Method("getClass",
```

Comment:
This shoud be wrapped in a `//@formatter:off`, `//@formatter:on` block because it's more clear on separate lines.